scripts/correctness.sh

```diff
@@ -102,5 +102,8 @@ python3 -m benchgc --verbose 0 --driver mlir --case ${CASE_DIR}/reduce.mlir || F
 # mlir
 # python3 -m benchgc --verbose 0 --driver mlir --case ${CASE_DIR}/llama2.mlir || FAIL=1
 
+#mlp
+python3 -m benchgc --verbose 1  --driver pattern --case mlp --batch_size=32 --hidden_size_list=32x16x64 --has_bias=1x1 --act_type=noop --dtype=f32 
```

Comment:
Is there any chance to merge `hidden_size_list` and `dtype`?
The command may look like `--hidden=32x16x64xf32 --has_bias=1x1xf32`?

Reply:
currently, we do not need a mlp pattern with different src/wei/bias dtypes, Let's leave this for now and make the changes later when needed.